OpenSim/Tools/ScaleTool.h

```diff
@@ -101,34 +101,43 @@ OpenSim_DECLARE_CONCRETE_OBJECT(ScaleTool, Object);
 #endif
     void copyData(const ScaleTool &aSubject);
 
-    Model* createModel();
+    Model* createModel() const;
     /* Query the subject for different parameters */
-    GenericModelMaker& getGenericModelMaker()
+    GenericModelMaker& getGenericModelMaker() const
     {
```

Comment:
IMHO if this method is `const` it should not provide non `const` access to a data member - in this case its a property and the `const` "contract" is easily violated. It is no better than being non-const in IMO.